t/reconfigure/consensus.t.js

```diff
@@ -32,9 +32,14 @@ function prove (async, assert) {
              -initial-cluster-state new', async())
     }, function () {
         var wait
-        var consensus = new Consensus(ip, '2379', function (properties, callback) {
-            wait(null, properties)
-            callback()
+        var consensus = new Consensus(ip, '2379', {
+            object: {
+                foo: function (properties, callback) {
+                    wait(null, properties)
+                    callback()
+                }
+            },
+            method: 'foo'
```

Comment:
No. Revert this. The point is that you can still use the `function` here. The interface didn't change, it just expanded.